atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -30,6 +31,22 @@
 from atomic_reactor.util import get_build_json, OSBSLogs, ImageName
 
 
+# an email address consisting of local name, an @ sign, and a domain name
+# the local name starts with a letter and then has one or more alphanumerics, _, +, and -
+# symbols in any order.
+# the domain name has one or more alphanumerics or - in any order
+# followed by a ., followed by one or more alphanumerics and - in any order.
+# RFC 2821 defines a valid email address much more expansively, but this definition matches
+# what most people loosely expect to be valid.
+# specifically, email addresses of the form id/name@domain.tla are not valid.
+VALID_EMAIL_REGEX = "^[a-zA-Z][a-zA-Z0-9-_.\+]+@[a-zA-Z0-9-\.]+\.[a-zA-Z0-9-]+$"
```

Comment:
Need to escape the `.` on the second range?

Consider using raw strings for defining regex: `r'....'`

Reply:
Fixed.